app/src/main/java/com/duckduckgo/app/httpsupgrade/api/HttpsUpgradeDataDownloader.kt

```diff
@@ -107,4 +113,22 @@ class HttpsUpgradeDataDownloader @Inject constructor(
 
     }
 
+    fun reportUpgradeStatistics(): Completable {
```

Comment:
We should consider renaming this class at some point, since it is no longer exclusively a `downloader` as the class name suggests - or move the `reportUpgradeStatistics` action to its own class?